edb/lib/net.edgeql

```diff
@@ -99,3 +99,38 @@ std::net::http::schedule_request(
         }
     ));
 };
+
+CREATE FUNCTION
+std::net::http::schedule_request(
+    url: str,
+    named only body: std::json,
+    named only headers: optional std::array<
+        std::tuple<
+            name: std::str,
+            value: std::str
+        >
+    > = {},
+    named only method: std::net::http::Method = std::net::http::Method.`GET`,
+) -> std::net::http::ScheduledRequest
+{
+    SET is_inlined := true;
+    USING ((
+        WITH 
+            has_content_type := any(
+                std::str_lower(std::array_unpack(headers).0) = 'content-type'
+            ),
```

Comment:
OK, apologies for the golfing, I just have a feeling like we're rarely going to have a chance to revisit this, so pays to put in a little effort now. 😅 

I wonder if this is better performance wise:

```suggestion
            has_content_type := (
                "content-type" in std::str_lower(std::array_unpack(headers).0)
            ),
```

@msullivan or @elprans have any insight here?

Not a blocker!